packages/devtools-connect/src/connect.spec.ts

```diff
@@ -429,20 +429,20 @@ describe('devtools connect', function () {
       ).to.equal(true);
     });
 
-    it('returns false if the PROVIDER_NAME JS option is set', function () {
+    it('returns false if the ENVIRONMENT JS option is set', function () {
       expect(
         isHumanOidcFlow('mongodb://example/?authMechanism=MONGODB-OIDC', {
           authMechanismProperties: {
-            PROVIDER_NAME: 'aws',
+            ENVIRONMENT: 'azure',
```

Comment:
Why did we change the value from aws to azure?

Reply:
Because `aws` is not actually a valid value for this 🙂 https://github.com/mongodb/specifications/blob/master/source/auth/auth.md#mongocredential-properties-6